pr_testing/_helper_functions.sh

```diff
@@ -196,3 +196,32 @@ function get_gpu_matrix_args() {
   OPTS=$(PYTHONPATH=${PR_TESTING_DIR}/.. ${CMSBOT_PYTHON_CMD} -c 'from RelValArgs import GPU_RELVALS_FLAGS;print(GPU_RELVALS_FLAGS)')
   echo ${OPTS}
 }
+
+function check_invalid_wf_lists () {
+  WKFS=$(echo $1 | sed -e 's/^-l//')
+  UPLOAD=${2:-true}
+  WFLISTS_CNT=$(echo "${WKFS}" | tr ',' '\n' | grep -v "^[1-9][0-9]*\(.[0-9][0-9]*\|\)\s" | wc -l)
+  WFS_CNT=$(echo "${WKFS}" | tr ',' '\n' | wc -l)
+  eval CMS_PATH=/cvmfs/cms-ib.cern.ch SITECONFIG_PATH=/cvmfs/cms-ib.cern.ch/SITECONF/$CMS_SITE_OVERRIDE runTheMatrix.py -j ${NJOBS:-1} -l ${WKFS} -n 2>&1 > tmp123.tmp
```

Comment:
@iarspider , may be batter to move `set +e ... set -e` around this command instead of every where we call check_invalid_wf_lists ?

Also only `runTheMatrix.py -l ${WKFS} -n` should work. `-n` does not run any jobs so no need to use `-j` and no need to set `CMS_PATH and SITECONFIG_PATH` env . These are only needed when on actually runs workflows

Reply:
> Sure, will remove those. For my own enlightment, why do we even use eval ... when running runTheMatrix [here](https://github.com/cms-sw/cms-bot/blob/66ed0340ca02a95d984185ba41a19b0adc74de87/run-ib-pr-matrix.sh#L103)? I thought that it will work the same without eval?

No idea, but I agree it should work without eval there too